core/pta/attestation.c

```diff
@@ -195,8 +195,6 @@ static TEE_Result deserialize_key(uint8_t *buf, size_t buf_sz)
 	sz = deserialize_bignum(p, buf_sz, key->n);
```

Comment:
Btw @jforissier, why are we not returning a `TEE_ERROR` code when `deserialize_bignum()` returns `0` ?